test/WebJobs.Script.Tests/TestScripts/Node/functions.tests.js

```diff
@@ -0,0 +1,132 @@
+﻿var expect = require('chai').expect;
+
+var config = process.argv[process.argv.length - 1]
+config = (config.indexOf('--config=') !== -1)? config.substr(9): 'Debug';
+
+var functions = require('../../bin/' + config + '/Content/Script/functions.js');
+
+var context = {};
+var logs = [];
+
+describe('functions', () => {
```

Comment:
Ah, I'll have to learn this describe/it this/it that test format. I'm used to the suite/test format :)